cmd/ansible-operator/main.go

```diff
@@ -144,13 +144,13 @@ func main() {
 		}
```

Comment:
Missing update the doc: https://github.com/operator-framework/operator-sdk/blob/1248c1ca2e0b7b7162471349daddb731657fe117/website/content/en/docs/building-operators/helm/reference/advanced_features.md#changing-the-concurrent-worker-count

Reply:
@varshaprasad96 @camilamacedo86 

This section was pre-emptively updated and wordsmith-ed here:

https://github.com/operator-framework/operator-sdk/pull/3326/files#diff-47162e7ecac88f3c6f2c91e49dcd7757R91-R105

It _would_ be better to make that update in this PR, so feel free to steal that section.

Reply:
merged the helm doc to help its updated. 
PS.: I update the doc and forgot :-) tks

Reply:
Made a small addition to the existing doc.